run-and-report.py

```diff
@@ -63,6 +79,7 @@ def command_name(command_array):
     parser.add_option("--tcp", default=False, action='store_true', help="Use TCP transport instead of UDP.")
     parser.add_option("--host", default=None, help="Specify hostname.")
     parser.add_option("--omit-metric", default=False, action='store_true', help="Do not send the metric.")
+    parser.add_option("--attributes", default=None, help="List of key=value attributes pairs")
```

Comment:
can you please make it clear that multiple attributes should be separated by commas?